src/pg/brc20/brc20-pg-store.ts

```diff
@@ -220,6 +222,50 @@ export class Brc20PgStore {
     });
   }
 
+  async getActivity(
+    args: {
+      ticker?: string[];
+      block_height?: number;
+      operation?: string[];
+    } & DbInscriptionIndexPaging
+  ): Promise<DbPaginatedResult<DbBrc20Activity>> {
+    const tickerConditions = this.sqlOr(
+      args.ticker?.map(t => this.sql`d.ticker_lower = LOWER(${t})`)
+    );
+
+    const results = await this.sql<(DbBrc20Activity & { total: number })[]>`
+      SELECT
+        e.operation,
+        d.ticker,
+        l.genesis_id AS inscription_id,
+        l.block_height,
+        l.block_hash,
+        l.tx_id,
+        l.address,
+        l.timestamp,
+        d.max AS deploy_max,
+        d.limit AS deploy_limit,
+        d.decimals AS deploy_decimals,
+        (SELECT amount FROM brc20_mints WHERE id = e.mint_id) AS mint_amount,
+        (SELECT amount || ';' || from_address || ';' || COALESCE(to_address, '') FROM brc20_transfers WHERE id = e.transfer_id) AS transfer_data,
+        COUNT(*) OVER() as total
```

Comment:
This count should only be a temporary measure for now. I believe it will cause us performance issues very quickly. We should think about caching some counts in a separate PR.